### PR TITLE
Feature/gutenberg

### PR DESCRIPTION
**Pull request by iruzevic**

### CODE REVIEW

**Comment by MirelaZ** [comment]
I'm not sure again I reviewed everything that has to be reviewed :) If I understand correctly, the Block section was deleted and doesn't need to be reviewed?

File: Gutenberg/General.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Now you can create blocks that you can preview and live edit from the backend without refreshing the frontend part of the website for every change.
+
+All development details can be found on:
+- [Github](https://github.com/WordPress/gutenberg/)
+- [Handbook](https://developer.wordpress.org/block-editor/)
+
+# Block types
+Gutenberg gives us the ability to create blocks that we are going to use on the frontend and the backend for the website. We have two kinds for a block in our deposal.
+
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply Gutenberg/General.md]: 👍

File: Gutenberg/General.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## 2. Dynamic blocks.
+This approach defines edit part to the block in javascript and frontend part in the PHP. Registration is mostly the same, but with some difference in the `save` method. In the `save` method, you put `return null,` and that tells Gutenberg to save only block comments with filled attributes data and not the HTML markup in the database.
+
+This approach is slower because you need to write the same markup in PHP and javascript, but it helps us overcome all the issues from the standard blocks.
+
+More development details can be found in [WordPress Handbook](https://developer.wordpress.org/block-editor/tutorials/block-tutorial/creating-dynamic-blocks/).
+
+## Documentation
+We are working with our custom build Eightshift Boilerplate Internal project. Wou will be working with these libs and should also read their Readme and Wiki materials:
+1. [Eightshift Boilerplate Internal](https://github.com/infinum/eightshift-boilerplate-internal)
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply Gutenberg/General.md]: 👍

File: Useful links/Useful links.md [hunk 3]
```diff
@@ -3,5 +3,13 @@
 [WordPress Core handbook](https://make.wordpress.org/core/handbook/)
 [WordPress Themes handbook](https://make.wordpress.org/themes/handbook/)
 [WordPress Plugins handbook](https://make.wordpress.org/plugins/handbook/)
+[Gutenberg documentation](https://developer.wordpress.org/block-editor/)
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply Useful links/Useful links.md]: 👍

**Comment by iruzevic** [comment]
@MirelaZ yes all the red parts are deleted so no reviews are necessary.

I have added the corrections if it is ok please approve this pull request. tnx

**Pull request merged**